helm-chart/renku/templates/graph/token-repository-secret.yaml

```diff
@@ -1,12 +0,0 @@
-apiVersion: v1
-kind: Secret
-metadata:
-  name: {{ template "renku.graph.tokenRepository.fullname" . }}
-  labels:
-    app: {{ template "renku.graph.tokenRepository.name" . }}
-    chart: {{ template "renku.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-type: Opaque
-data:
-  tokenRepository-tokenEncryption-secret: {{ required "Fill in .Values.graph.tokenRepository.tokenEncryption.secret with `openssl rand -hex 8|base64`" .Values.graph.tokenRepository.tokenEncryption.secret | b64enc | quote }}
```

Comment:
@olevski FYI this is why these secrets are currently double-encoded: we ask the admin to generate the secret with: `openssl rand -hex 8|base64` but then the template base64 encodes that **again**.

Reply:
Wow that is wild. I never noticed that.